manifests/generate.sh

```diff
@@ -96,15 +112,18 @@ done
 # Then do a bit of cleanup to reduce the directory depth to 1.
 ##########################################################################
 ${HELM} template \
-	-n tigera-operator \
-	../charts/tigera-operator/ \
-	--output-dir ocp \
-	--no-hooks \
-	--set installation.kubernetesProvider=OpenShift \
-	--set installation.enabled=false \
-	--set apiServer.enabled=false \
-	--set tigeraOperator.version=$OPERATOR_VERSION \
-	--set calicoctl.tag=$CALICO_VERSION
+  -n tigera-operator \
+  ../charts/tigera-operator/ \
+  --output-dir ocp \
+  --no-hooks \
+  --set installation.kubernetesProvider=OpenShift \
+  --set installation.enabled=false \
+  --set apiServer.enabled=false \
+  --set tigeraOperator.image=$OPERATOR_IMAGE \
+  --set tigeraOperator.version=$OPERATOR_VERSION \
+  --set tigeraOperator.registry=$OPERATOR_REGISTRY \
+  --set calicoctl.image=$REGISTRY/ctl \
```

Comment:
Seems wrong that `calicoctl.image` is of the form REGISTRY/CTL whereas the `tigeraOperator.image` is just of the form `tigera/operator` with the registry configured separately - we should be using consistent meanings for image / version / registry.

Reply:
the calicoctl images are pushed as <registry>/ctl

sadly we currently do not follow a consistent pattern b/w operator and monorepo images, that will require a bigger overhaul which is beyond the scope of this PR

Reply:
> The calicoctl images are pushed as /ctl

Surely they must be pushed as `docker.io/calico/ctl` no?

Operator has two separate config options - `registry` and `image`

Calicoctl has a single option - `image` - that appears to be a merged version of registry and image.

Reply:
For my dev images, it is created as `docker.io/tuticat/ctl`

When the default registry is not overwritten, this be set to `calico/ctl` which matches the current state of `image: calico/ctl:master`